### PR TITLE
Hw2: Tusk 2.2

### PR DESCRIPTION
**Pull request by DmitriySmertin**

### CODE REVIEW

File: src/hw2.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Output: {"a": 1, "d": 2, "e": 1, "g": 2}
+"""
+from typing import Dict, List
+
+
+def transform(legacy_data: Dict[int, List[str]]) -> Dict[str, int]:
+    new_dict = dict()
+    for i in legacy_data:
+        for k in legacy_data.get(i):
+            new_dict.update({k: i})
```
Comment (alpakhomov) [inline src/hw2.py]: You can change letter case right here:
```py
new_dict.update({k.lower(): i})
```
And then you won't need low_dict variable and the third loop

**Review by alpakhomov — APPROVED** [review]
Left a comment & approved

File: src/hw2.py (hunk 1 above)
Comment (DmitriySmertin) [reply src/hw2.py]: Thx

**Review by alpakhomov — APPROVED** [review]
Approved!

**Pull request merged**